wagtail/images/utils.py

```diff
@@ -94,7 +95,9 @@ def generate_signature(image_id, filter_spec, key=None):
 
 
 def verify_signature(signature, image_id, filter_spec, key=None):
-    return force_str(signature) == generate_signature(image_id, filter_spec, key=key)
+    return constant_time_compare(
```

Comment:
was going to mention that signature comes is encoded to bytes, whereas `generate_signature` returns a string, however `constant_time_compare` forces both values to bytes (TIL), so it is all good